Introduction to Kotlin Multiplatform/06_jvm_render.md

```diff
@@ -102,12 +119,12 @@ actual fun Colors.newColor(r: Int, g: Int, b: Int) = Color(r, g, b)
 actual val Colors.BLACK: Color get() = Color.BLACK
 ```
  
-By these declarations we say that our expected class `Color` means
-`java.awt.Color` on the JVM. We also provide implementations for
+By these declarations we say that our expected class `Color`
+should be `java.awt.Color` class from the JVM. We also provide implementations for
 two more required functions.
 
 We should be able to start the JVM application now. Let's check
```

Comment:
We should be able to start the JVM application now. Let's check it by running the Gradle `run` task. We should see similar output in the console